internal/providers/terraform/google/service_networking_connection.go

```diff
@@ -0,0 +1,22 @@
+package google
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+)
+
+func getServiceNetworkingConnectionRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "google_service_networking_connection",
+		RFunc: newServiceNetworkingConnection,
+	}
+}
+
+func newServiceNetworkingConnection(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	region := d.Get("region").String()
+	return &schema.Resource{
+		Name: d.Address,
+		SubResources: []*schema.Resource{
+			networkEgress(region, u, "Network egress", "Traffic", ComputeVPNGateway),
```

Comment:
We could copy this function to `internal/resources/google` for now and introduce a new resource there. It may help someone who'll be migrating the other related resources.

Reply:
Yeah I considered that and I think we should but I noticed that @sinabakh has already started that process recently (~2 weeks) so I figured I see how that plays out.

P.S. Thanks for the networkEgress work/refactor @sinabakh, it made adding this resource super easy!